docs/competency-api/documentation.md

```diff
@@ -5,4 +5,94 @@ nav_order: 1
 parent: Condition
 grand_parent: Competency API
 ---
-# Documentation
\ No newline at end of file
+# Documentation
+
+## Description
+
+### Headers for all requests
+```json
+{
+    Bearer: "Bearer Example.bearer.token"
+}
+```
+
+### Create
+
+#### post:
+```http
+https://api.competency-constructor.clark.center/competencies/:competencyId/documentation
+```
```

Comment:
Post route missing request body, example, and status codes

Reply:
they are the same so I meant to have the request act for both patch and update, I can make that more obvious